torch/hub.py

```diff
@@ -111,32 +112,32 @@ def _get_torch_home():
 
 def _parse_repo_info(github):
     if ':' in github:
-        repo_info, branch = github.split(':')
+        repo_info, ref = github.split(':')
```

Comment:
I renamed (private) uses of the term "branch" into "ref", as this is more accurate. This is somewhat orthogonal to the relevant changes made in this PR, so I can revert if it makes the review difficult.